packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -542,16 +549,17 @@ export async function DocsQA<Doc extends Document>(props: DocsQAProps<Doc>) {
   if (status !== Corpus.LoadingState.COMPLETED) {
     return `Corpus is not loaded. It's in state: ${status.toString()}`;
   }
-  const docs = await props.corpus.search(props.question);
+  const docs = await props.corpus.search(props.question, { limit: props.limit });
   return (
     <ChatCompletion>
       <SystemMessage>
-        You are a customer service agent. Answer questions truthfully. Here is what you know:
+        You are a trained question answerer. Answer questions truthfully, using only the document excerpts below. Do not
```

Comment:
You can replace some of this with `<Prompt persona='trained question answerer' noInternalKnowledge />`.

Reply:
yeah, I'm not a fan of that particular pattern, I don't think it offers enough to make up for the additional complexity. I think we probably want to consider some other ways of approaching this sort of "prompt reuse".

Reply:
Legit. Looking forward to seeing how things develop and whether the pattern ultimately feels useful.